### PR TITLE
iframe compatibility

### PR DESCRIPTION
**Pull request by acting326**
- Uses brandcast branch for iframe compatibility
- Editor reference updated for a non-shadowdom-wrapped crx

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "brace": "^0.10.0",
     "draft-convert": "github:appcues/draft-convert",
-    "draft-js": "github:appcues/draft-js",
+    "draft-js": "brandcast/draft-js-built#003f24b",
```
Comment (rayd) [inline package.json]: Does this depend on React 16?

Comment (acting326) [reply package.json]: @rayd I thought there would be issues there, but when I installed only the draft-js branch, the only dependency issue was with immutable.  If it's running smoothly, does that mean it's 👍 ?

Comment (rayd) [reply package.json]: I guess so! We'll have to do some thorough testing

Comment (rayd) [reply package.json]: We should update the version of immutable specified in `package.json` -- I don't _love_ the idea of running such a brand new (not even fully released) version of immutable, but if we test everything thoroughly, I'm okay with it.

**Pull request merged**